src/reducers/__tests__/__snapshots__/reducers-test.js.snap

```diff
@@ -26,10 +26,12 @@ Object {
   "notifications": Object {
     "general": true,
   },
-  "projectList": Object {},
-  "projectWorkflows": Object {},
+  "projectList": Array [],
+  "projectListHolding": Array [],
```

Comment:
What is the purpose of `projectListHolding`? You mentioned it briefly above, but I was curious in general.